snapcraft/internal/repo/_platform.py

```diff
@@ -26,6 +26,7 @@
     '"elementary"',
     'debian',
     'neon',
+    'LinuxMint',
```

Comment:
For my case code has been work. 
Test python script has return "LinuxMint".
File /etc/os-release has string "ID=linuxmint".
Who should I believe?